### ISSUE TITLE
suggest layout ideas

### ISSUE THREAD

**Issue by 0xf17** [design-futures/design-futures.github.io#2]
How might we arrange your projects/inquiries on the landing page?
The plan is to have one landing page and then separate pages for each of your project/inquiry. Each project may contain a title, abstract (150-200 words), 3-5 hires images.

**Comment by Stawrman** [comment]
We could have a landing page with some visuals surrounding design and futures, maybe even a design embedded in the webpage itself. For example the landing page could have some sort of a mirror, that inverts the left side of the page to the right division
Just wondering aloud

**Comment by 0xf17** [comment]
Great. Can we have some rough sketches to understand how you are imagining?

**Comment by VikashRaj715** [comment]
Di you want to discuss @Stawrman

**Comment by Stawrman** [comment]
cool

**Comment by Akshata-20196** [comment]
We can create a layout with some brief about the studio Design and Future on left side of page and on right side we can create links for the project of students. Just an idea.

**Comment by Stawrman** [comment]
https://drive.google.com/drive/folders/15NsVdyFGt977ZhpS6p1fqYipZf5YtD63?usp=sharing

**Comment by Stawrman** [comment]
Hi Gaurav, Vikas and others please have a look^

**Comment by Stawrman** [comment]
So the idea is straightforward, we divide our understanding into two sides. The left side, our ideas, assumptions, imagination. The right side is what we actually made. And both on top and bottom we could have some sort of marquees for the questions and inquiries we were following

**Comment by thewiseguyshivam** [comment]
I made a possible layout that could be considered, the idea is to keep the website simple and clean so the focus is on the subjects and the projects:
[https://ujwaltrust925-my.sharepoint.com/:b:/g/personal/205331025_srishtimanipalinstitute_in/Eepvti1-2WpLhpzwBnlLiKIBvnz9HV_olqUSCPTn328y3A?e=luARl8](url)

**Comment by thewiseguyshivam** [comment]
I made a possible layout to be considered, the idea is to keep the website clean and simple, so that the focus is on the subjects and the projects:
[Layoutidea1.pdf](https://github.com/design-futures/design-futures.github.io/files/5479596/Layoutidea1.pdf)

**Comment by 0xf17** [comment]
Both of the ideas look interesting. Continue fleshing them out and discuss here. You can start converging into a single concept as we move ahead in the day.

**Comment by Stawrman** [comment]
Hmm yeah

**Comment by Stawrman** [comment]
On second thoughts we can also use some of studio project ideas as a string to guide the website
We've got a cuckoo nest, braille, minecraft,, an underwater city etc. etc.

**Comment by Stawrman** [comment]
Again, we can think of simplifying all of this later

**Comment by Akshata-20196** [comment]
I am putting my possible idea about layout. Please have a look and let me know
![Untitled-2](https://user-images.githubusercontent.com/73871506/97957650-a7789500-1dd1-11eb-8027-bc7b011cf52c.jpg)

**Comment by Stawrman** [comment]
Thanks for sharing Akshata, what if we placed the circle in the centre, moved the intro down below the circle?

**Comment by Stawrman** [comment]
Also, I prefer straightforward, behance-like thumbnails because they give a better and direct visual feedback on everyone's work. Is there a way we can incorporate that into the pie or the circle?

**Comment by 0xf17** [comment]
We can get away from the nav menu. Do not really need this simple website. Also, I think the right section can take up the center stage here, the studio abstract can be reduced to a smaller space on the page. The pie chart idea looks on the similar lines to what Shivam suggested. We could use something like https://observablehq.com/@d3/hierarchical-edge-bundling to make it more interesting. 

@Akshata-20196 Do you want to start messing around with html/js to prototype this idea?

**Comment by Stawrman** [comment]
This version of the pie network looks fantastic

**Comment by Stawrman** [comment]
There's also some Sunburt pie recommendations under that, I think those are suitable for the relatively simpler nature of our projects

**Comment by thewiseguyshivam** [comment]
That edge bundling idea is really cool, we can use it to link projects based on subjects maybe.
We could also add details on the participants of this studio maybe and link them based on interest

**Comment by Stawrman** [comment]
Think this one works^

**Comment by VikashRaj715** [comment]
On a simpler note, I was thinking of having a way to show a preview of the work for the student listing.
<img width="740" alt="Screenshot 2020-11-03 at 12 52 54 PM" src="https://user-images.githubusercontent.com/73871395/97958767-e7407c00-1dd3-11eb-8fa8-404e9555b0df.png">

**Comment by VikashRaj715** [comment]
Will it be possible to do in the pie-chart style?

**Comment by 0xf17** [comment]
A few of you can start exploring edge bundling and similar visualizations at https://vega.github.io/editor/#/examples/vega/edge-bundling. 

Others may continue to throw in more ideas. We will need to find ways to converge ideas after that.

**Comment by Stawrman** [comment]
hm. The pie chart is now a bundle even idk. The only thing is, our work is divided into stages. will the bundle be able to convert that?

**Comment by Stawrman** [comment]
Like the ideation, conception, prototypes, podcast etc etc. the whole hierarchy

**Comment by 0xf17** [comment]
<img width="646" alt="Screenshot 2020-11-03 at 12 59 53 PM" src="https://user-images.githubusercontent.com/2363826/97959013-7a79b180-1dd4-11eb-8451-6f812d0101f6.png">
edge bundling may be rearranged in this way to show hierarchy. But I'm keen on keeping each project/inquiry page to minimum as this will be in public domain. We may let go off podcasts and other deliverables from this website, unless some of you want to. And keep only curated content on this website but not all. Thoughts?

**Comment by RuchiJain750** [comment]
![c4f87ec2-d332-4dd7-a6f2-ec2c5e9cfaec](https://user-images.githubusercontent.com/73874287/97959236-e9570a80-1dd4-11eb-862d-66bfd02d03a8.jpg)

**Comment by thewiseguyshivam** [comment]
Yeah I agree, not everyone will want all the stuff there, I think we can ask the participants to mark their submissions on whether they are comfortable with it going on the website or not.

**Comment by Stawrman** [comment]
but then what are we going to make the bundle off of?

**Comment by 0xf17** [comment]
once we lock in the layout and concept, we can identify specific requirements for the project page and share with everyone to give us that information. Bundle could be made either by connecting your inquiries with each other or with the keywords that we generate from each of your inquiry.

**Comment by Akshata-20196** [comment]
Yes Gaurav, I can start but I am not sure about how I am going to do that pie chart stuff

**Comment by Stawrman** [comment]
Okay Gaurav

**Comment by RuchiJain750** [comment]
![Uploading f4ff8bc7-b100-49ed-aace-3fbe8da9f923.jpg…]()

**Comment by 0xf17** [comment]
Ruchi, idea of a logo is nice. Can you and others who may be interested in ideating and sketching out a logo that may capture the essence of studio? 

Akshata, You may experiment it here: https://vega.github.io/editor/#/examples/vega/edge-bundling. But you could prototype with P5.JS or bare JS itself, as it is comfortable to you. It doesn't need to exactly like it. That was merely a suggestion. @Stawrman @thewiseguyshivam may join you as well. 

A simplest version could be even hand drawn sketch with certain portions hyperlinked.

**Comment by Akshata-20196** [comment]
Okay Gaurav. I will start looking it and will try to make something out of that

**Comment by RuchiJain750** [comment]
Okay. Sure

**Comment by thewiseguyshivam** [comment]
@RuchiJain750 I'm interested in working on the logo

**Comment by blueegems** [comment]
![Uploading C28F9347-CB88-4FD2-B9CE-7E5ED140EBED.jpeg…]()

**Comment by RuchiJain750** [comment]
![Presentation1](https://user-images.githubusercontent.com/73874287/97962154-9718e800-1dda-11eb-80ca-5696867d486d.png)

**Comment by blueegems** [comment]
![image0(2)](https://user-images.githubusercontent.com/66700480/97962568-7c933e80-1ddb-11eb-822f-0de42f057e7e.jpeg)

If we are going for a circular sorta theme here, this could be an option

**Comment by Stawrman** [comment]
Hey anybody drafted a logo or something?

**Comment by thewiseguyshivam** [comment]
Ruchi and I are on it

**Comment by Abinaya-Nagappan** [comment]
![bubbles](https://user-images.githubusercontent.com/66700480/97964833-68e9d700-1ddf-11eb-9d81-563c99db787b.jpeg)

We could have interactive bubbles just like our venn diagrams which would lead to our projects

**Comment by Stawrman** [comment]
thewiseguyshivam could you guys share what you're doing here

**Comment by Abinaya-Nagappan** [comment]
![inytigur2](https://user-images.githubusercontent.com/66700480/97964921-8f0f7700-1ddf-11eb-9ef7-08981c8c3c78.jpeg)

Or just intriguing 10 word brief of our enquiry to make the random user curious

**Comment by Stawrman** [comment]
oooo, lez screw with their hEAd

**Comment by 0xf17** [comment]
Like your idea Abinaya. 

Now that we have many ideas on the table. It will be good if you can work in pairs/groups towards more specific parts for the concept. 
1. Landing page
- Logo (Ruchi/Shivam are currently working on this)
- Layout
- Information/Content
2. Project/Inquiry page
- What is the information we require from each to go into these pages?
- Layout
3. Overall visual look

Self delegate and form pairs/groups to work on one or few aspects from above. I'm not sure how many of you can also start prototyping with html/js here except Akshata here. So maybe Akshata can work on the layout for prototype landing page and others may join her. Others who want to give a shot at html may start explore the landing page. Thoughts?

Is anyone keen to explore visual/typography/look and feel for the website? Will be good to have some ideas on that front as well.

**Comment by Abinaya-Nagappan** [comment]
I can be on the team that handles information requirements to go into individual pages. It'll be great to have a couple more people on this! :D

**Comment by Stawrman** [comment]
landing page content & info

**Comment by 0xf17** [comment]
Perfect! Once you form pairs/groups. Post a message here with details of what you can take care of, with names of your group members. Self assign tasks and proceed.

**Comment by Abinaya-Nagappan** [comment]
> 
> 
> landing page content & info

Ok Stawrman, lets work together! btw whats your real name? 😅

**Comment by Akshata-20196** [comment]
I am working on prototype.

**Comment by Stawrman** [comment]
Naveen Bagalkot

**Comment by Abinaya-Nagappan** [comment]
> 
> 
> Naveen Bagalkot

OH! hello sir.. sorry 🙈😂

**Comment by RuchiJain750** [comment]
![79da58cb-967a-4b7f-a9d3-c7dd32ff17c6](https://user-images.githubusercontent.com/73874287/97970120-0ac0f200-1de7-11eb-8f01-cf90a8baeb11.jpg)

**Comment by RuchiJain750** [comment]
just 1st sketch of some logo ideas. I have tried to show a sense of direction with eyes indication speculation

**Comment by sudhanshu-rastogi** [comment]
how about a deck of cards played out for the lay out 
![deck-playing-cards-laid-out-green-table-deck-playing-cards-laid-out-green-table-top-card-ace-spades-192903712](https://user-images.githubusercontent.com/73881926/97972205-2d083f00-1dea-11eb-921c-80add913fc36.jpg)

**Comment by VikashRaj715** [comment]
https://www.pantheoneaudio.com/

https://wtower.ru/#en

https://atelier.net/virtual-economy/

https://bartlettarchucl.com/ss20

https://hellotoday.jp/

Sharing a couple of website links that have various kind of typography and visual styles. We can discuss and pick ideas from them if it helps!

**Comment by naveen-bl** [comment]
Just jumping in to say, amazing stuff! Whether we will have anything to exhibit or not, we shall have a great place to exhibit!

**Comment by Akshata-20196** [comment]
![image](https://user-images.githubusercontent.com/73871506/97974769-18c64100-1dee-11eb-895d-afe17651598e.png)
I have designed this layout in HTML and CSS. Please take a look and let me know

**Comment by 0xf17** [comment]
Good start. Arranging them in the grid doesn't make it look like the Venn diagrams as I m assuming you are working on Abinaya's initial idea and also it would be difficult to divide categories in mutually exclusive way.

Others working on layout, leave your suggestions.

**Comment by Akshata-20196** [comment]
Yes, I make the separate bubbles according to the categories through which we make the peer groups.

**Comment by thewiseguyshivam** [comment]
So for the logo what Ruchi and I decided on until now is this:
1. We'll put up a Form, and everyone (students/facilitators) will be required to draw and put a simple line drawing/shape (black on white background) based on what they interpret speculative design as.
2. We'll have a boundary shape for the logo, and each time the website is opened, one of those 25(?) line drawings will be put in that boundary, and then it will be overlaid with a random set of colors, which will be sort of thrown over the line drawing, imagine throwing liquid ink on a drawing.

That will make the logo sort of unpredictable and dynamic while still having a structure, and we think it captures the spirit of the studio pretty well?

Please share your thoughts on this!

**Comment by 0xf17** [comment]
@thewiseguyshivam That sounds great! Get started on the Form, we don't have much time left for this studio and the website. @Others add your thoughts so we can speed into building the website.

**Comment by naveen-bl** [comment]
> So for the logo what Ruchi and I decided on until now is this:
> 
> 1. We'll put up a Form, and everyone (students/facilitators) will be required to draw and put a simple line drawing/shape (black on white background) based on what they interpret speculative design as.
> 2. We'll have a boundary shape for the logo, and each time the website is opened, one of those 25(?) line drawings will be put in that boundary, and then it will be overlaid with a random set of colors, which will be sort of thrown over the line drawing, imagine throwing liquid ink on a drawing.
> 
> That will make the logo sort of unpredictable and dynamic while still having a structure, and we think it captures the spirit of the studio pretty well?
> 
> Please share your thoughts on this!

Sounds great! 
get us the forms quickly!

**Comment by RuchiJain750** [comment]
Update : We have made the form and uploaded it on teams.

**Comment by Abinaya-Nagappan** [comment]
> 
> 
> Good start. Arranging them in the grid doesn't make it look like the Venn diagrams as I m assuming you are working on Abinaya's initial idea and also it would be difficult to divide categories in mutually exclusive way.
> 
> Others working on layout, leave your suggestions.

Hey yes Gaurav! Mutual exclusivity is an issue in that, we are all working on some better landing page ideas as well. 

> 
> 
> Yes, I make the separate bubbles according to the categories through which we make the peer groups.

Yep noted @0xf17 
We'll all get working to provide better wireframes to @Akshata-20196 and make it hopefully dazzzle. (hopefully)

**Comment by Abinaya-Nagappan** [comment]
> 
> 
> Yes, I make the separate bubbles according to the categories through which we make the peer groups.

Yep noted @0xf17
We'll all get working to provide better wireframes to @Akshata-20196 and make it hopefully dazzzle. (hopefully)

**Comment by Aayush-x404x** [comment]
Hey guys this looks nice. I'm sorry I can't contribute as of now because I have a lot on my plate but do let me know if I can help in anyway.
